doc/dnf5_plugins/automatic.8.rst

```diff
@@ -0,0 +1,238 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _automatic_plugin_ref-label:
+
+##################
+ Automatic command
+##################
+
+Synopsis
+==========
+
+``dnf5 automatic [options] [<config file>]``
+
+
+Options
+=======
+
+``--timer``
+    Apply random delay before execution.
+
+``--downloadupdates``
+    Automatically download updated packages.
+
+``--no-downloadupdates``
+    Do not automatically download updated packages.
+
+``--installupdates``
+    Automatically install downloaded updates (implies --downloadupdates).
+
+``--no-installupdates``
+    Do not automatically install downloaded updates.
+
+
+Arguments
+=========
+
+``<config file>``
+    Path to configuration file. Defaults to ``/etc/dnf/automatic.conf``.
+
+
+Description
+=============
+
+Alternative CLI to ``dnf upgrade`` with specific facilities to make it suitable to be executed automatically and regularly from systemd timers, cron jobs and similar.
```

Comment:
Note: I believe that it is an alternative not only t ``dnf upgrade``, but also to ``check-updates``

Reply:
Well, it kind of is. But I still consider documenting it like an `upgrade` alternative clearer. Automatic have basically three modes:

1. only check - this is like `dnf upgrade` when the user aborts the transaction
2. download upgrades - this is like `dnf upgrade --downloadonly`
3. and finally do the upgrades - alternative to `dnf upgrade` or `dnf upgrade --security`